### ISSUE TITLE
Movement stops as soon as I stop moving my finger on the trackpad.

### ISSUE THREAD

**Issue by kakek** [KozGit/DOOM-3-BFG-VR#256]
Hello all.

Thanks for this awesome mod. I'm amazed it's working this well, and iy turns out doom 3 looks pretty nice in VR. I'm looking forward toward replaying it this way.

Now my issue : I have to constantly move my finger on the trackpad to keep moving. And I do mean move, and not just keep it on the trackpad. It's like the trackpad only detect my finger while it's moving.
This is obviously very onnoying and imprecise, as I have to keep making small movement of my thumb on the upper area of the trackpad to walk.

I am using latest steamVR beta and a WMR headset and controlers. Everything else seems to be working perfectly. The right controler doesn't even behave this way (if I keep my finger on the trackpad, the weapons keep switching.

Is this an option I missed / misunderstood ?

EDIT : 
Some precisions. This seems to be linked to left controler, and not movement in itself. Since it even apply in menus. If I keep my finger on right trackpad, the option keep changing. If I do the same on the left controller, it changes only once.
Also, obviously tried in other VR games, and it only happens in this one.

Thanks in advance for any help.

NB : Also, would be nice to have thumbstick support, since WMR controler do have them. But I know that's a separate feature.

**Comment by Violins77** [comment]
I have the same thing with the Samsung Odyssey. I did try to implement analog stick movement, but I had a hard time compiling it and didn't have time to dig into it since. Good news is, you can now emulate spatial movement directly with Mixed Reality!

https://www.reddit.com/r/WindowsMR/comments/7xey62/anyone_else_see_the_new_thumbstick_controls_in/
